admin/zabbix/Makefile

```diff
@@ -72,13 +74,21 @@ endef
 define Package/zabbix-server
   $(call Package/zabbix/Default)
   TITLE+= server
-  DEPENDS += +pgsql-cli +libevent2
+  DEPENDS += +mysql-server +libmariadb +libevent2
+endef
+
+define Package/zabbix-server-frontend
+  $(call Package/zabbix/Default)
+  TITLE+= server-frontend
+  DEPENDS += +zabbix-server +php7 +php7-cgi +php7-fpm +php7-mod-mysqli \
+  +php7-mod-gd +php7-mod-bcmath +php7-mod-ctype +php7-mod-xmlreader +php7-mod-xmlwriter \
+  +php7-mod-session +php7-mod-sockets +php7-mod-mbstring +php7-mod-gettext
 endef
 
 define Package/zabbix-proxy
   $(call Package/zabbix/Default)
   TITLE+= proxy
-  DEPENDS += +pgsql-cli
+  DEPENDS += +mysql-server +libmariadb
```

Comment:
again, this seems out of scope.  And are you sure you need teh _server_ to run the proxy?  (note that pgsql-cli doesn't pull in a pgsql server)

Reply:
Proxy it is a buffer with local database which sends data to the server.
It also gets dependency to local database.
[1].https://www.zabbix.com/documentation/3.4/manual/concepts/proxy